Makefile

```diff
@@ -31,9 +31,6 @@ LOG_LEVEL ?= debug
 # Image URL to use all building/pushing image targets
 IMG ?= hive-controller:latest
 
-# Image to use when deploying
-DEPLOY_IMAGE ?= registry.ci.openshift.org/openshift/hive-v4.0:hive
```

Comment:
IMG is now used for make deploy as well, but I have removed the default registry CI image. Deploying from source should now be developers only, and should never need to go to the CI image. Normal users should use OLM, otherwise they will need to build their own image.